### PR TITLE
Disable graphics

### PR DESCRIPTION
**Pull request by tinayang3024**
#### Description
- add NO_GRAPHICS flags to prevent including ezgl library when graphics are disabled
- does not compile ezgl library if gtk/x11 are not installed

#### Related Issue

#878 

#### How Has This Been Tested?

- built & ran on machine with no gtk/x11 installed
- built & ran on machine with gtk/x11 installed

#### Types of changes

- [x] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by mithro** [comment]
PTAL @acomodi

File: vpr/src/base/vpr_api.cpp [hunk 1]
```diff
@@ -845,8 +841,10 @@ void vpr_init_graphics(const t_vpr_setup& vpr_setup, const t_arch& arch) {
 }
 
 void vpr_close_graphics(const t_vpr_setup& /*vpr_setup*/) {
+#ifndef NO_GRAPHICS
```
Comment (kmurray) [inline vpr/src/base/vpr_api.cpp]: Looks like this is the only one left out side of vpr/draw. Can it also be removed?

**Comment by kmurray** [comment]
Thanks @tinayang3024!

I think aside from the one last #ifndef noted above, the only thing left is to resolve the conflicts with the search bar changes (i.e. merge or rebase your branch with master). After that it should be good to merge.

**Comment by kmurray** [comment]
@tinayang3024 Thanks for finishing this!

**Comment by tinayang3024** [comment]
> @tinayang3024 Thanks for finishing this!

Thank you for taking the time to review my code

**Pull request merged**